pysatCDAAC/instruments/cosmic_gps.py

```diff
@@ -104,6 +113,78 @@ def init(self):
     return
 
 
+def clean(self):
+    """Return COSMIC GPS data cleaned to the specified level.
+
+    Parameters
+    ----------
+    self : pysat.Instrument
+        Instrument class object, whose attribute clean_level is used to return
+        the desired level of data selectivity.
+
+    Notes
+    -----
+    Supports 'clean', 'dusty', 'dirty'
+
+    """
+    if self.tag == 'ionprf':
+        # ionosphere density profiles
+        if self.clean_level == 'clean':
+            # try and make sure all data is good
+            # filter out profiles where source provider processing doesn't
+            # get max dens and max dens alt
+            self.data = self.data[((self['edmaxalt'] != -999.) &
+                                   (self['edmax'] != -999.))]
+            # make sure edmaxalt in "reasonable" range
+            self.data = self.data[(self.data.edmaxalt >= 175.) &
+                                  (self.data.edmaxalt <= 475.)]
```

Comment:
The style of accessing data should be consistent.  I think line 136 is recommended.

Reply:
This one wasn't addressed in the last commit.

Reply:
It shows up as changed in my code.  :confused:

Reply:
Oops, misread this

Reply:
Yeah, you fixed the think I didn't notice, but also needed fixing :P. I looked at the new changes and you only did the part I highlighted, not the rest of the file where this style issue persists.  You may have already addressed it, but noting in case you were being quick and forgot.

Reply:
Only found one other line to fix.  If I'm missing something else, let's discuss via a chat